perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2313,10 +2313,17 @@ public Member setSponsoredMember(PerunSession session, Vo vo, User userToBeSpons
 		PasswordManagerModule module = getPerunBl().getUsersManagerBl().getPasswordManagerModule(sess, namespace);
 
 		for (String name : names) {
-			// generate random password
-			String password = module.generateRandomPassword(sess, null);
 			Map<String, String> mapName = new HashMap<>();
-			mapName.put("guestName", name);
+			if (name.contains(";")) {
```

Comment:
You must also add this way of parsing to the javadoc (Bl, entry, RPC and also openapi).

Reply:
I have updated the documentation.